<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brew Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
   <addaction name="actionAdd_Step"/>
   <addaction name="actionRemove_Step"/>
   <addaction name="actionUpload_Recipe"/>
   <addaction name="actionStart_Brewing"/>
   <addaction name="actionNext_Step"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>&amp;Calls</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionClear"/>
    <addaction name="actionAdd_Step"/>
    <addaction name="actionRemove_Step"/>
    <addaction name="actionUpload_Recipe"/>
    <addaction name="actionStart_Brewing"/>
    <addaction name="actionNext_Step"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd_Step">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Step</string>
   </property>
   <property name="toolTip">
    <string>Adds a new brew step</string>
   </property>
  </action>
  <action name="actionUpload_Recipe">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/upload.png</normaloff>:/images/upload.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Upload Recipe</string>
   </property>
   <property name="toolTip">
    <string>Send the Upload brew recipe over the serial port</string>
   </property>
  </action>
  <action name="actionStart_Brewing">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start Brewing</string>
   </property>
   <property name="toolTip">
    <string>Starts the brew process</string>
   </property>
  </action>
  <action name="actionNext_Step">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next Step</string>
   </property>
   <property name="toolTip">
    <string>Go to the next brew step</string>
   </property>
  </action>
  <action name="actionRemove_Step">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Step</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected brew step</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;ve</string>
   </property>
   <property name="toolTip">
    <string>Save the brew recipe</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="brewMaster.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a brew recipe</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="brewMaster.qrc"/>
 </resources>
 <connections/>
</ui>
